c10/core/Stream.h

```diff
@@ -1,6 +1,13 @@
 #pragma once
 
 #include <c10/core/Device.h>
+#include <c10/core/DeviceType.h>
+#include <c10/macros/Export.h>
+#include <c10/util/Exception.h>
+#include <cstddef>
+#include <cstdint>
+#include <functional>
+#include <ostream>
```

Comment:
Do we need ostream for all these headers? Or would iosfwd be sufficient?

Reply:
ostream functions are used in the header.